support/reattach-pv/README.md

```diff
@@ -1,21 +1,32 @@
 # Reattach-PV
 
-This tool can be used to recreate an Elasticsearch cluster by reusing orphaned PersistentVolumes that used to belong to that cluster before it was deleted.
+This tool can be used to recreate an Elasticsearch cluster by reusing orphaned PersistentVolumes that used to belong to a cluster before it was deleted.
 
 **Warning**: to be used at your own risk. This tool has not been tested extensively with multiple Kubernetes distributions and PersistentVolume providers. You should backup the data in the underlying storage system before attempting to use this tool. Also make sure you perform a dry-run first.
 
 ## Expectations
 
 This tool can only be used when the following conditions are met:
 
+1. If re-building a cluster that was deleted, and no longer exists.
+
 * The Elasticsearch resource to re-create does not exist in Kubernetes.
 * All PersistentVolumeClaims of the previous cluster do not exist anymore.
 * All PersistentVolumes of the previous cluster still exist with the status `Released`.
 * The Elasticsearch resource to re-create has the exact same specs as the deleted one. Same cluster name, same node sets, same count, etc.
 * The current default kubectl context targets the desired Kubernetes cluster.
 
+2. If building a new cluster, with a new name, from existing unused PVs from a previously deleted, and re-created cluster
+
+* The Elasticsearch resource to create does not exist in Kubernetes. (The previous cluster with the previous name can exist, using new PVs)
+* All PersistentVolumes of the previous cluster still exist with the status `Released`.
+* The Elasticsearch resource to create has the exact same specs (same node sets, same count, etc.) as the deleted, and re-created one, but with different cluster name.
+* The current default kubectl context targets the desired Kubernetes cluster.
+
```

Comment:
I think the shorter the doc, the more likely it is to be read. I find it a bit hard to read. It took me a while to understand the differences between the two. 

Only `* All PersistentVolumeClaims of the previous cluster do not exist anymore` doesn't apply for the second case. We could rephrase it in `* All PersistentVolumeClaims to re-create does not exist anymore` but I think `* The Elasticsearch resource to re-create does not exist in Kubernetes.` covers that.

So we could just keep one list of conditions without the one for the PersistentVolumeClaims and just mention that the cluster name can be the same or not?

```suggestion
* The Elasticsearch resource to re-create does not exist in Kubernetes.
* All PersistentVolumes of the deleted cluster still exist with the status `Released`.
* The Elasticsearch resource to re-create has the exact same nodeSet specifications as the deleted one (same nodeSet names and counts).
* The current default kubectl context targets the desired Kubernetes cluster.

The Elasticsearch resource to be recreated can have the same name as the deleted one or a new name. In the second case, you must provide the name of the deleted cluster through the flag `--old-elasticsearch-name`.

```